Documentation/ApiOverview/TSFE/Index.rst

```diff
@@ -0,0 +1,216 @@
+.. include:: /Includes.rst.txt
+.. index:: TSFE; TypoScriptFrontendController
+.. _tsfe:
+
+====
+TSFE
+====
+
+What is TSFE?
+=============
+
+TSFE is short for :php:`\TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController`,
+a class which exists in the system extension EXT:frontend.
+
+As the name implies: A responsibility of TSFE
+is page rendering. It also handles reading from and writing to the page cache.
+For more details it is best to look in the source code.
+
+There are several contexts in which the term TSFE is used:
+
+1.  PHP: It is passed as request attribute
+    :ref:`frontend.controller <typo3-request-attribute-frontend-controller>`
+2.  PHP: It was and is available as global array :php:`$GLOBALS['TSFE']` in PHP.
+3.  TypoScript: TypoScript function :ref:`TSFE <t3tsref:data-type-gettext-tsfe>`
+    which can be used to access public properties in TSFE.
+4.  (deprecated since v9.5 and :doc:`removed in 10.0
+    <ext_core:Changelog/10.0/Breaking-88564-PageTSconfigSettingTSFEconstantsRemoved>`)
+    Page TSconfig: :typoscript:`TSFE.constants`.
+
+Focusing on the PHP part (as the TypoScript part is covered in the
+:ref:`TypoScript Reference: TSFE <t3tsref:data-type-gettext-tsfe>` page),
+this page gives an overview, what can still be used, what is deprecated,
```

Comment:
```suggestion
this page gives an overview, what can be used, what is deprecated,
```
The "still" implies (at least for me), that TSFE is phased-out (which is not true AFAIK).